cmd/kpod/create.go

```diff
@@ -80,20 +80,21 @@ type createConfig struct {
 	groupAdd           []uint32          // group-add
 	hostname           string            //hostname
 	image              string
-	interactive        bool              //interactive
-	ip6Address         string            //ipv6
-	ipAddress          string            //ip
-	labels             map[string]string //label
-	linkLocalIP        []string          // link-local-ip
-	logDriver          string            // log-driver
-	logDriverOpt       []string          // log-opt
-	macAddress         string            //mac-address
-	name               string            //name
-	network            string            //network
-	networkAlias       []string          //network-alias
-	nsIPC              string            // ipc
-	nsNET              string            //net
-	pidMode            container.PidMode //pid
+	interactive        bool                  //interactive
+	ip6Address         string                //ipv6
+	ipAddress          string                //ip
+	labels             map[string]string     //label
+	linkLocalIP        []string              // link-local-ip
```

Comment:
nit of a nit, gotta space after // in the comment for these 3, but not in the others.  Yes my OCD is kicking in.

Reply:
I actually think we should just remove these comments, since they were just a helper to @baude to remember the link between the CLI option and the internal storage